### DISCUSSION TITLE
== Error ==  path contains non-ASCII characters: /mnt/DiscoSeaGate_3Tb/PROYECTOS_GERMAN/XXXX_seq/11304/assembly/ (output directory)

### DISCUSSION THREAD

**Discussion by germant13** [ablab/spades#748] [Q&A]
Hello!
I have an error when I tried to run SPades software
The name of server hard disk contain a non-ASCII character (" _ "). 
What can I do to solve the error?

Regards

**Comment by asl** [comment]
Hello,

Consider using other directory for output.

**Comment by germant13** [comment]
I tried with different directory, but the problem continued...
The main problem is the name of the hard disk mounted. 
are there any manner to skip this kind of non-ASCII character?

**Comment by asl** [comment]
Well, remount the disk under different mount point?
